packages/node_modules/@webex/plugin-meetings/src/meeting/index.js

```diff
@@ -2303,6 +2303,10 @@ export default class Meeting extends StatelessWebexPlugin {
           }
           if (this.config.stats.enableStatsAnalyzer) {
             this.statsAnalyzer = new StatsAnalyzer(this.config.stats);
+            this.statsAnalyzer.on(EVENT_TRIGGERS.MEDIA_QUALITY, (res) => {
+              // TODO:  might have to send the same event to the developer
+              Metrics.postEvent({event: eventType.MEDIA_QUALITY, meetingId: this.correlationId, data: {intervalData: res}});
```

Comment:
Yes, I would like to emit this event to developers as well

Reply:
ya we need to change the data structure a bit. I dont know if they will understand it . We need to filter out so many data there . So a separate task

Reply:
Please remove the TODO line above.